File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/NameChangePacket.scala
package net.server.channel.packet.cash.operation

class NameChangePacket(private var _action: Int, private var _itemId: Int, private var _oldName: String, private var _newName: String) extends BaseCashOperationPacket(_action = _action) {
  def itemId: Int = _itemId

  def oldName: String = _oldName

  def newName: String = _newName
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/NameChangePacket.java
package net.server.channel.packet.cash.operation;

public class NameChangePacket extends BaseCashOperationPacket {
   private final Integer itemId;

   private final String oldName;

   private final String newName;

   public NameChangePacket(Integer action, Integer itemId, String oldName, String newName) {
      super(action);
      this.itemId = itemId;
      this.oldName = oldName;
      this.newName = newName;
   }

   public Integer itemId() {
      return itemId;
   }

   public String oldName() {
      return oldName;
   }

   public String newName() {
      return newName;
   }
}
